src/e2e-test/java/io/cdap/plugin/bigquerymultitable/locators/BigQueryMultiTableLocators.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.bigquerymultitable.locators;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+/**
+ * BigQueryMultiTable Related Locators.
+ */
+public class BigQueryMultiTableLocators {
+
+  @FindBy(how = How.XPATH, using = "//input[@data-cy='datasetProject']")
+  public static WebElement datasetProject;
+
+  @FindBy(how = How.XPATH, using = "//input[@data-cy='location']")
+  public static WebElement location;
```

Comment:
Where are we using these 2 locators I don't see them in actions.

Reply:
removed